app-editors/ghostwriter/files/install-fix-1.8.1.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ghostwriter.pro b/ghostwriter.pro
+index d399fb5..7f5f22c 100644
+--- a/ghostwriter.pro
++++ b/ghostwriter.pro
+@@ -264,7 +264,7 @@ macx {
+     desktop.path = $$DATADIR/applications/
+ 
+     appdata.files = resources/linux/ghostwriter.appdata.xml
+-    appdata.path = $$DATADIR/appdata/
++    appdata.path = $$DATADIR/metainfo/
+ 
+     man.files = resources/linux/ghostwriter.1
+     man.path = $$PREFIX/share/man/man1
```

Comment:
Is that patch coming from or submitted upstream?

Reply:
Yesterday I opened a PR to upstream with that patch :)

Reply:
Excellent - could you then use the commit you submitted upstream to create that patch? so that it includes a proper header.

Use, in your ghostwriter.git clone, `git format-patch HEAD^1 --stdout > <that-FILESDIR>/ghostwriter-1.8.1-appdata-path.patch`